Makefile

```diff
@@ -71,6 +71,7 @@ CORE_MAINTAINER?=	project@opnsense.org
 CORE_ORIGIN?=		opnsense/${CORE_NAME}
 CORE_PACKAGESITE?=	https://pkg.opnsense.org
 CORE_PRODUCT?=		OPNsense
+CORE_PRODUCT_LEGAL?=OPNsense®
```

Comment:
what have I done ;( shuffling UTF-8 through make and json is not so great

Reply:
It seems to handle it correctly, though.

Reply:
If people go through the trouble of rebranding they can zap the (r) in the volt or you make it conditional as well. This is too little benefit to have it included and opnsense-version doesn't know about it.

Reply:
I discussed this approach with Jos and he preferred it this way.

I think we can use UTF-8 in Makefiles anno 2018. As I mentioned, I did test it and it works correctly.

Reply:
ok fine, I disagree with having this piggybacked through a build system

Reply:
It's no worse than having our website URL piggybacked through it, IMO. As for zapping the ® manually: I find that very inconsistent with your initial request of using template variables. Either we should use template variables throughout or require manual editing, not a mix.

I will change the approach, but I heavily disagree: I find your stance _very_ inconsistent.

Reply:
You don't have to change it for me. Ad can merge it when he is back. I just don't see a point in passing both "OPNsense" and "OPNsense(r)" for a single use in a file that has no functional use. Moreover, the user neither sees nor cares about the change so why do it in the first place? If you feel like I'm being inconsistent it is because you're trying to argue if it's already inconsistent it's only consistent to keep building on the inconsistencies. :)

Reply:
No, I felt you were being inconsistent because we already have a lot of stuff that isn't _strictly_ make-related (such as URLs) in our makefile. To me, adding something to that should be fine.

I changed it because I asked Jos again and he said that, since you felt strongly about it and you handle most of the build stuff, it would probably be better to follow your instructions. So I did.

There are a lot of unreviewed, unmerged PRs already -- I don't really want to stall this one just because I disagree with you over what, in the end, isn't that major. I'm taking your approach, and I will complain no further about it.

Reply:
Look, take a bit of time to look at the history of the code you're changing if in doubt. The previous location for the product data was worse. And I'd replace this current one in a heartbeat if there was a better idea. So all I am asking is not starting to clutter stuff that is not great in an of itself. I'm fully aware...

Thing is I would have not started reviewing if that change was in the original PR or said no directly. The purpose of the PR is not to sneak in more later because that derails the review process unless it's agreed that such a change is necessary. The other change was great and it was correct to take that risk, but one good and one bad change still stalls the PR.

Reply:
You can thumbs down this all you like, because my point still stands. :)

Reply:
You requested using more template variables, I discussed what approach I should take with Jos and then applied it. That is _not_ sneaking stuff. You can accuse me of plenty of things, but not of "sneaking stuff it into PRs", and I don't appreciate this one bit.

Reply:
I'm sorry for phrasing it badly. In the scope of this PR, previously undiscussed changes were added that inadvertently altered the review state in my opinion. I can only judge this PR. I'm not judging you.

Reply:
Ok, thanks!